### PR TITLE
Updated czech translation

### PR DESCRIPTION
**Pull request by Tajnymag**
Hi, I'm not the best programmer, but I'd like to help you at least with
translating the app to czech. I looked at the values and realized, that
almost a half of the original english stuff is untranslated or missing.
I fixed that and translated everything I could :-)

### CODE REVIEW

File: cSploit/res/values-cs/strings.xml [hunk 1]
```diff
@@ -346,39 +365,125 @@
...
         <item>MSF jenom kompatibilní </item>
         <item>jen na exploitdb </item>
-        <item>ze dvou exploitdb a MSF</item>
+        <item>z obou exploitdb i MSF</item>
     </string-array>
     <string-array name="pref_performance_exploit_values">
         <item>MSF</item>
         <item>EXDB</item>
-        <item>oboje</item>
+        <item>OBOJE</item>
```
Comment (tux-mind) [inline cSploit/res/values-cs/strings.xml]: this value should be unchanged: "BOTH"

File: cSploit/res/values-cs/strings.xml [hunk 2]
```diff
@@ -346,39 +365,125 @@
     <string-array name="pref_performance_vuln_values">
         <item>OSVDB</item>
         <item>CVE</item>
-        <item>oboje</item>
+        <item>OBOJE</item>
```
Comment (tux-mind) [inline cSploit/res/values-cs/strings.xml]: this value should be unchanged: "BOTH"

**Comment by tux-mind** [comment]
@cSploit/translators some issue with values that should be in English ( see my comments on commits ).

@Tajnymag thanks for your previous help, really appreciated :blush:

**Comment by tux-mind** [comment]
merged in 8adad43 :blush:

**Comment by Tajnymag** [comment]
Thanks :-) I'll be careful next time. :+1:

**Pull request closed without merging**